db/migrate/20200324130825_change_date_to_timestamp.rb

```diff
@@ -0,0 +1,198 @@
+class ChangeDateToTimestamp < ActiveRecord::Migration
+  def up
+    safety_assured do
+      change_column :access_tokens, :created_at, :datetime
+      change_column :access_tokens, :updated_at, :datetime
+      change_column :accounts, :created_at, :datetime
+      change_column :accounts, :updated_at, :datetime
+      change_column :accounts, :paid_at, :datetime
+      change_column :accounts, :deleted_at, :datetime if column_exists?(:accounts, :deleted_at)
```

Comment:
Yeah ... somehow Postgres has this column already removed ... 🤦‍♂

Reply:
I know, I am dealing with that in the columns removal PR :joy:

Reply:
Anyway that column is not used at all anymore so it wouldn't be a drama we don't change the column type for this one.

Reply:
Btw we see from this list some tables and columns that are not used anymore, candidates for the big removal of columns.